ocaml/xapi-cli-server/cli_frontend.ml

```diff
@@ -3684,6 +3684,15 @@ let rec cmdtable_data : (string * cmd_spec) list =
       ; flags= []
       }
     )
+  ; ( "repository-introduce-remote-pool"
+    , {
+        reqd= ["name-label"; "binary-url"; "certificate"]
```

Comment:
Are we expecting the certificate as a string on the CLI? How long is this string? Would it not be more natural to expect a file containing the certificate?